### ISSUE TITLE
Purge dependency on script-editor

### ISSUE THREAD

**Issue by ctrueden** [scijava/scripting-python#2]
This component has a dependency on `org.scijava:script-editor` so that it can implement a `PythonScriptSyntaxHighlighter`. But this drags in several other dependencies including scijava-ui-swing, which is undesirable. The syntax highlighting layer should probably be its own layer. We need to scrutinize the modularity of the script-editor component to improve the architecture here.

**Comment by karlduderstadt** [comment]
Yes, I remember worrying about this and trying to think of an alternative approach. One quick fix could be to move PythonScriptSynataxHghlighter to the highlighters package in the script-editor. https://github.com/scijava/script-editor/tree/9deba3a1f4dcac109aec6f93a2265b3a587aa859/src/main/java/org/scijava/ui/swing/script/highliters

As long as the name matches it should work and that would remove the dependency issue. But obviously that is perhaps not the most elegant solution.
